plugins/filter/to_prettytable.py

```diff
@@ -0,0 +1,382 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2025, Timur Gadiev <tgadiev@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+
+DOCUMENTATION = '''
+  name: to_prettytable
```

Comment:
Also, for consistency with other plugins, please do not indent the top level of the documentation blocks.
```suggestion
name: to_prettytable
```
(replicate throughout this and the other doc blocks, please)

Reply:
Done